solr/contrib/ltr/src/test/org/apache/solr/ltr/TestRerankBase.java

```diff
@@ -402,7 +401,7 @@ protected static void bulkIndex(String filePath) throws Exception {
       h.updater.handleRequest(req, res);
     } catch (final Throwable ex) {
       // Ignore. Just log the exception and go to the next file
-      logger.error(ex.getMessage());
+      log.error(ex.getMessage());
```

Comment:
I think that ``log.error(ex.getMessage(), ex)`` will add the stack trace in the logs